crates/solidity/outputs/cargo/crate/Cargo.toml

```diff
@@ -35,8 +35,8 @@ default = ["cli"]
 cli = ["dep:anyhow", "dep:clap", "dep:serde_json"]
 
 [dependencies]
-anyhow = { workspace = true, optional = true }
 ariadne = { workspace = true }
```

Comment:
nit: I wonder why was that changed? we usually keep these alphabetically sorted.

Reply:
mixed up trying to fix deps with napi. fixed